src/Numerics/tests/CMakeLists.txt

```diff
@@ -24,13 +24,14 @@ set(UTEST_SRCS
     test_transform.cpp
     test_min_oned.cpp
     test_OneDimCubicSplineLinearGrid.cpp
-    test_one_dim_cubic_spline.cpp)
+    test_one_dim_cubic_spline.cpp
+    test_Quadrature.cpp)
 
 # Run gen_gto.py to create these files.  They may take a long time to compile.
 #SET(UTEST_SRCS ${UTEST_SRCS} test_full_cartesian_tensor.cpp test_full_soa_cartesian_tensor.cpp)
 
 add_executable(${UTEST_EXE} ${UTEST_SRCS})
 
-target_link_libraries(${UTEST_EXE} catch_main containers qmcio)
+target_link_libraries(${UTEST_EXE} catch_main containers qmcio qmcwfs)
```

Comment:
Is qmcwfs really needed?

Reply:
the internal test in quadrature calls SoaSphericalTensor to do its checks. SoaSphericalTensor is in the qmcwfs library, so that was the only was I could get it to compile without undefined symbol errors

Reply:
if there is a simpler solution let me know

Reply:
How about move SoaSphericalTensor.h/cpp under Numerics?

Reply:
that should work. Will give it a go

Reply:
refactored SoaSphericalTensor and SoaCartessianTensor into Numerics. The Quadrature test doesn't use the SoaCartesianTensor, but it seemed like it should be in Numerics as well